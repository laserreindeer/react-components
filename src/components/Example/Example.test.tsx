import { render } from "@testing-library/react";
import { Example } from "./Example";

it("matches snapshot loading", () => {
	const { asFragment } = render(<Example loading>Test</Example>);
	expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot loaded", () => {
	const { asFragment } = render(<Example>Test</Example>);
	expect(asFragment()).toMatchSnapshot();
});
