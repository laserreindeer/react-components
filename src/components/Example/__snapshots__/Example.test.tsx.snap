// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot loaded 1`] = `
<DocumentFragment>
  <div>
    Test
  </div>
</DocumentFragment>
`;

exports[`matches snapshot loading 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
